pkgs/applications/audio/fverb/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, faust2lv2 }:
```

```suggestion
{ stdenv
, fetchFromGitHub
, faust2lv2
}:
```
Minor formatting, optional